src/Farmer/Arm/ServiceBus.fs

```diff
@@ -109,6 +109,14 @@ module Namespaces =
                            enablePartitioning = this.EnablePartitioning |> Option.toNullable |}
                 |} :> _
 
+module private Sku =
```

Comment:
I would suggest making this a member on the `Sku` type (consistency).

Reply:
I think I fixed this here. https://github.com/CompositionalIT/farmer/pull/625